res/TensorFlowLiteRecipes/Net_StridedSlice_StridedSlice_000/test.recipe

```diff
@@ -0,0 +1,92 @@
+operand {
+  name: "ifm"
+  type: INT32
+  shape { dim: 1 dim: 2 dim: 4 dim: 4 }
+}
+operand {
+  name: "begin"
+  type: INT32
+  shape { dim: 4 }
+  filler {
+    tag: "explicit"
+    arg: "1" arg: "1" arg: "1" arg: "1"
+  }
+}
+operand {
+  name: "end"
+  type: INT32
+  shape { dim: 4 }
+  filler {
+    tag: "explicit"
+    arg: "1" arg: "2" arg: "4" arg: "4"
+  }
+}
+operand {
+  name: "strides"
+  type: INT32
+  shape { dim: 4 }
+  filler {
+    tag: "explicit"
+    arg: "1" arg: "1" arg: "1" arg: "1"
+  }
+}
+operand {
+  name: "output"
+  type: INT32
+  shape { dim: 1 dim: 2 dim: 4 dim: 4 }
+}
+operation {
+  type: "StridedSlice"
+  strided_slice_options {
+    begin_mask: 0
+    end_mask: 0
+    ellipsis_mask: 0
+    new_axis_mask: 0
+    shrink_axis_mask: 0
+  }
+  input: "ifm"
+  input: "begin"
+  input: "end"
+  input: "strides"
+  output: "output"
+}
+operand {
+  name: "begin_2"
+  type: INT32
+  shape { dim: 4 }
+  filler {
+    tag: "explicit"
+    arg: "0" arg: "0" arg: "0" arg: "0"
+  }
+}
+operand {
+  name: "end_2"
+  type: INT32
+  shape { dim: 4 }
+  filler {
+    tag: "explicit"
+    arg: "0" arg: "1" arg: "0" arg: "0"
```

Comment:
this value looks something weird. this value not matching output shape.

Reply:
Done